src-ui/src/app/components/document-detail/document-detail.component.html

```diff
@@ -34,11 +34,25 @@
         </svg>&nbsp;<span class="d-none d-lg-inline" i18n>More like this</span>
     </button>
 
-    <button type="button" class="btn btn-sm btn-outline-primary" (click)="close()">
+    <button type="button" class="btn btn-sm btn-outline-primary me-2" i18n-title title="Close" (click)="close()">
         <svg class="buttonicon" fill="currentColor">
             <use xlink:href="assets/bootstrap-icons.svg#x" />
         </svg>&nbsp;<span class="d-none d-lg-inline" i18n>Close</span>
```

Comment:
```suggestion
        </svg>
```

Reply:
Sorry forgot this one, to make the close / prev / next buttons all icon-only. Thanks!

Reply:
perfect. good idea to remove the text and make them all icon buttons, I wasn't entirely happy with the solution either, this is cleaner.